coalib/bearlib/abstractions/Linter.py

```diff
@@ -316,18 +319,28 @@ def _convert_output_regex_match_to_result(self,
                 groups['origin'] = '{} ({})'.format(klass.__name__,
                                                     groups['origin'].strip())
 
-            # Construct the result.
-            return Result.from_values(
-                origin=groups.get('origin', self),
-                message=(groups.get('message', '').strip()
-                         if result_message is None else result_message),
-                file=filename,
-                severity=groups['severity'],
-                line=groups['line'],
-                column=groups['column'],
-                end_line=groups['end_line'],
-                end_column=groups['end_column'],
-                additional_info=groups.get('additional_info', '').strip())
+            # Construct the result. For LocalBears we use Result.from_values
+            # and give all the file related info.
+            # For GlobalBears we consruct a project scope result.
+            if filename is not None:
```

Comment:
I like to check for None-ness explicitly like this. This way the reader understands quickly that I'm only checking for None-ness at this point and don't handle empty strings or `0`